libultraship/libultraship/SDLController.cpp

```diff
@@ -441,7 +450,7 @@ namespace Ship {
     }
 
     const char* SDLController::GetControllerName() {
-        return SDL_GameControllerNameForIndex(physicalSlot);
+        return strdup(ControllerName.c_str());

```

Comment:
Is the `strdup` necessary?
The pointer returned by `c_str` is valid as long as `ConrollerName` is not destroyed.

Reply:
Yes